src/ch/ehi/ili2db/fromili/TypeUtility.java

```diff
@@ -0,0 +1,172 @@
+package ch.ehi.ili2db.fromili;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+
+import ch.ehi.ili2db.base.Ili2dbException;
+import ch.ehi.ili2db.base.DbNames;
+import ch.ehi.ili2db.dbmetainfo.DbExtMetaInfo;
+import ch.ehi.ili2db.mapping.NameMapping;
+
+public class TypeUtility {
+
+    public static void writeColumnProps(Connection conn, String dbSchema, NameMapping mapping)
+        throws Ili2dbException {
+        /**
+         * Writes in the meta-info table the possible values of the t_type
+         * column, comma separated.
+         */
+
+        ArrayList<String> tables = getTablesWithTypeSetting(conn, dbSchema);
+        for (String table : tables) {
+            writeColumnProp(conn, dbSchema, table, getSubClasses(conn, dbSchema, mapping.mapSqlTableName(table)));
+        }
+    }
+
+    private static void writeColumnProp(Connection conn, String dbSchema, String table, ArrayList<String> subClasses)
```

Comment:
do not update the t_ili2dd meta tables directly to avoid incomplete --createScript generated sql scripts. Just populate the DbExtMetaInfo instance